ext/gd/libgd/gd.h

```diff
@@ -624,6 +628,13 @@ gdImagePtr gdImageCreateFromGif(FILE *fd);
 gdImagePtr gdImageCreateFromGifCtx(gdIOCtxPtr in);
 gdImagePtr gdImageCreateFromGifSource(gdSourcePtr in);
 
+//TODO: we may not need all of these
```

Comment:
I think this comment can just be dropped; even if we don't need some of these functions, it's better to have them for easier sync with external libgd.

Reply:
makes sense! Keeping functions, dropping comment.